pkg/state/filter_test.go

```diff
@@ -307,4 +307,44 @@ routes:
 			Expect(returnedState).To(MatchYAML(filteredState))
 		})
 	})
+	Context("when there is a linux bridge without 'bridge' options because is down", func() {
+		BeforeEach(func() {
+			state = nmstate.NewState(`
+interfaces:
+- name: br1
+  type: linux-bridge
+  state: down
+  ipv4:
+    enabled: false
+  ipv6:
+    enabled: false
+  lldp:
+    enabled: false
+  mac-address: A2:EE:84:7B:42:4C
+  mtu: 1500
```

Comment:
None of this is needed to test that you fixed the issue, is it?

Reply:
I have replicate what we have in the env that is failing the bridge is like this.

Reply:
But then it is not a test for missing bridge. It is a test for missing bridge *and* another issues that we haven't thought about and may be in the code. We may catch a bug by chance here, but it feels really random and non-fitting for this specific test. You may argue that we should include snapshot of the whole state of a very complicated node to catch more potential issues

Reply:
But we want to test with sane data right ? this attributes are present at bridge like this when a bridge is down.

Reply:
Ok, will just keep name, type and state.

Reply:
You are writting a unit test. The test should be concerned only about a specific subpart of the system. If we need the whole state to be populated to get one specific function running, then we have an issue in our architecture and end-up with ton of unmaintainable yamls.

We had a bug with missing bridge. You should write a minimal reproducer and then a fix.

If you expect that e.g. "lldp" may affect run of the function, then we need a test for invalid lldp.